integration_tests/CMakeLists.txt

```diff
@@ -219,4 +219,5 @@ RUN(NAME test_bit_length     LABELS cpython llvm)
 
 RUN(NAME generics_01         LABELS cpython llvm)
 RUN(NAME generics_array_01   LABELS llvm)
+RUN(NAME generics_list_01    LABELS cpython)
```

Comment:
Should we add `llvm` here?

Reply:
Added it. The generic mean is working now.